pkg/flowaggregator/s3uploader/s3uploader_test.go

```diff
@@ -108,7 +95,10 @@ func TestCacheRecord(t *testing.T) {
 }
 
 func TestBatchUploadAll(t *testing.T) {
-	mockS3Uploader := &mockS3Uploader{testReader: &bytes.Buffer{}}
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+	mockS3Uploader := s3uploadertesting.NewMockS3UploaderAPI(ctrl)
+	mockS3Uploader.EXPECT().Upload(context.TODO(), gomock.Any(), nil).Return(nil, nil)
```

Comment:
would it make sense to match the actual `PutObjectInput` parameter in the expectation (instead of using `gomock.Any()`)?

Reply:
https://github.com/antrea-io/antrea/blob/fd1188730208339412c3a81d389068bacb37dfa1/pkg/flowaggregator/s3uploader/s3uploader.go#L340
value of field `Key` of `PutObjectInput` consists of a random suffix, which is generated inside of function `Upload`
https://github.com/antrea-io/antrea/blob/fd1188730208339412c3a81d389068bacb37dfa1/pkg/flowaggregator/s3uploader/s3uploader.go#L316
value of field `Body` is initialized in function `batchUploadAll` 


maybe I'm wrong but it's not very straight-forward to me about how to get the expected `PutObjectInput` parameter in advance?

Reply:
yes, no problem. I was just checking with you.